chain/types/src/lib.rs

```diff
@@ -1,44 +1,34 @@
 //! This crate contains various on-chain related modules and types.
 
-use ethers::prelude::*;
-use hex_literal::hex;
+use alloy::{contract::Result as ContractResult, network::TransactionBuilder, primitives};
 use serde::{Deserialize, Serialize};
-use std::sync::Arc;
+use utils::ERC_1820_REGISTRY_DEPLOY_CODE;
 
-use hopr_bindings::hopr_announcements::HoprAnnouncements;
-use hopr_bindings::hopr_channels::HoprChannels;
-use hopr_bindings::hopr_dummy_proxy_for_network_registry::HoprDummyProxyForNetworkRegistry;
-use hopr_bindings::hopr_network_registry::HoprNetworkRegistry;
-use hopr_bindings::hopr_node_management_module::HoprNodeManagementModule;
-use hopr_bindings::hopr_node_safe_registry::HoprNodeSafeRegistry;
-use hopr_bindings::hopr_node_stake_factory::HoprNodeStakeFactory;
-use hopr_bindings::hopr_safe_proxy_for_network_registry::HoprSafeProxyForNetworkRegistry;
-use hopr_bindings::hopr_ticket_price_oracle::HoprTicketPriceOracle;
-use hopr_bindings::hopr_token::HoprToken;
-use hopr_bindings::hopr_winning_probability_oracle::HoprWinningProbabilityOracle;
+use hopr_bindings::{
+    hoprannouncements::HoprAnnouncements::{self, HoprAnnouncementsInstance},
```

Comment:
The module generation without the underscore is rather badly readable, is there a way how to get the underscore back in?

Reply:
It depends on the outputs of `forge`. Unless we fork it and modify their macro, the auto-gen bindings are camelCased, like in the smart contracts.

Reply:
Well it's also automated created by forge. I've created an issue regarding their lagging of using the latest alloy-rs in binding generation macros. Will create another issue on the naming of files and types.